test_runner/fixtures/zenith_fixtures.py

```diff
@@ -1890,6 +1890,7 @@ def try_start(self):
         with open(os.path.join(self.datadir, "etcd.log"), "wb") as log_file:
             args = [
                 self.binary_path,
+                f"--quota-backend-bytes=100000000",
```

Comment:
Out of curiosity, what's that for?

Reply:
Oh, that was supposed to be another PR. Let me do that and explain there :-).

Reply:
https://github.com/neondatabase/neon/pull/1791